opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+name: "eq"
+version: "0.1"
+maintainer: "Jonas Enlund <jonas.enlund@gmail.com>"
+author: "Jonas Enlund <jonas.enlund@gmail.com>"
+homepage: "https://github.com/jonase/eq"
+license: "MIT"
+bug-reports: "https://github.com/jonase/eq/issues"
+dev-repo: "git://github.com/Leonidas-from-XIV/slacko"
```

Comment:
Serves me right for copy-pasting :-( Thanks for spotting.

Will force push a fix when I get home.